<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drib</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MainWindow
{
background-image: url(&quot;code.jfif&quot;);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string> калькулятор для дробів</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>268</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ділення                 |</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>віднімання             |</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="add">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>обернений дріб 2   |</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>додавання             |</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="obern_2">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>обернений дріб 1   |</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>множення              |</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="divide">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="down1">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="multiply">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="down2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="subs">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="obern_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTextEdit" name="up1">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="up2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="adddd">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Додавання</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subbb">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Віднімання</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divvv">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ділення</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mulll">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Множення</string>
    </property>
   </widget>
   <widget class="QPushButton" name="obern_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Обернений перший</string>
    </property>
   </widget>
   <widget class="QPushButton" name="obern_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Обернений другий</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
